src/fsharp/FSharp.Compiler.Service.netcore/project.json

```diff
@@ -0,0 +1,298 @@
+    "../../absil/ilwrite.fsi",
+    "../../absil/ilwrite.fs",
+    "../../absil/ilreflect.fs",
+    "../../utils/CompilerLocationUtils.fs",
+    "../PrettyNaming.fs",
+    "../../ilx/ilxsettings.fs",
+    "../../ilx/EraseClosures.fsi",
+    "../../ilx/EraseClosures.fs",
+    "../../ilx/EraseUnions.fsi",
+    "../../ilx/EraseUnions.fs",
+    "../UnicodeLexing.fsi",
+    "../UnicodeLexing.fs",
+    "../layout.fsi",
+    "../layout.fs",
+    "../ast.fs",
+    "pppars.fs",
+    "pars.fs",
+    "../lexhelp.fsi",
+    "../lexhelp.fs",
+    "pplex.fs",
+    "lex.fs",
+    "../LexFilter.fs",
+    "../tainted.fsi",
+    "../tainted.fs",
+    "../ExtensionTyping.fsi",
+    "../ExtensionTyping.fs",
+    "../QuotationPickler.fsi",
+    "../QuotationPickler.fs",
+    "../tast.fs",
+    "../TcGlobals.fs",
+    "../TastOps.fsi",
+    "../TastOps.fs",
+    "../TastPickle.fsi",
+    "../TastPickle.fs",
+    "../import.fsi",
+    "../import.fs",
+    "../infos.fs",
+    "../NicePrint.fs",
+    "../AugmentWithHashCompare.fsi",
+    "../AugmentWithHashCompare.fs",
+    "../NameResolution.fsi",
+    "../NameResolution.fs",
+    "../TypeRelations.fs",
+    "../PatternMatchCompilation.fsi",
+    "../PatternMatchCompilation.fs",
+    "../ConstraintSolver.fsi",
+    "../ConstraintSolver.fs",
+    "../CheckFormatStrings.fsi",
+    "../CheckFormatStrings.fs",
+    "../FindUnsolved.fs",
+    "../QuotationTranslator.fsi",
+    "../QuotationTranslator.fs",
+    "../PostInferenceChecks.fsi",
+    "../PostInferenceChecks.fs",
+    "../TypeChecker.fsi",
+    "../TypeChecker.fs",
+    "../Optimizer.fsi",
+    "../Optimizer.fs",
+    "../DetupleArgs.fsi",
+    "../DetupleArgs.fs",
+    "../InnerLambdasToTopLevelFuncs.fsi",
+    "../InnerLambdasToTopLevelFuncs.fs",
+    "../LowerCallsAndSeqs.fs",
+    "../autobox.fs",
+    "../IlxGen.fsi",
+    "../IlxGen.fs",
+    "../CompileOps.fsi",
+    "../CompileOps.fs",
+    "../CompileOptions.fsi",
+    "../CompileOptions.fs",
+    "../fsc.fsi",
+    "../fsc.fs",
+    "../vs/IncrementalBuild.fsi",
+    "../vs/IncrementalBuild.fs",
+    "../vs/Reactor.fsi",
+    "../vs/Reactor.fs",
+    "../vs/ServiceConstants.fs",
+    "../vs/ServiceDeclarations.fsi",
+    "../vs/ServiceDeclarations.fs",
+    "../vs/Symbols.fsi",
+    "../vs/Symbols.fs",
+    "../vs/Exprs.fsi",
+    "../vs/Exprs.fs",
+    "../vs/ServiceLexing.fsi",
+    "../vs/ServiceLexing.fs",
+    "../vs/ServiceParseTreeWalk.fs",
+    "../vs/ServiceNavigation.fsi",
+    "../vs/ServiceNavigation.fs",
+    "../vs/ServiceParamInfoLocations.fsi",
+    "../vs/ServiceParamInfoLocations.fs",
+    "../vs/ServiceUntypedParse.fsi",
+    "../vs/ServiceUntypedParse.fs",
+    "../vs/service.fsi",
+    "../vs/service.fs",
+    "../vs/SimpleServices.fsi",
+    "../vs/SimpleServices.fs",
+    "../fsi/fsi.fsi",
+    "../fsi/fsi.fs"
+  ],
+  "dependencies": {
+    "Microsoft.FSharp.Core.netcore": "1.0.0-*",
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0-rc2-3002426"
```

Comment:
if you open `project.json.lock` and search for `Microsoft.NETCore.App`, you see it reference some packages. NETCore.App is an empty package, just reference other packages.
You can directly use the depednencies of NETCore.App inside the project.json, it's the same.
After that, we can remove unused dependencies (like csharp and vb) and others.

I think the real list is https://github.com/Microsoft/visualfsharp/blob/master/src/fsharp/FSharp.Compiler.netcore.nuget/Microsoft.FSharp.Compiler.netcore.nuspec
Fixing the -rc2 build should be enough